src/db/messages/mod.rs

```diff
@@ -0,0 +1,11 @@
+use crate::db::models;
+use actix_web::actix::*;
```

Comment:
Do we need actix here?

Reply:
We do. `actix` imports the prelude which has the `Message` trait. We need `actix_web` for the `actix_web::Error` type.

Reply:
Ok, I made it more explicit. I also updated a few other places I was importing `actix` or `actix_web` wildcards.